graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/generators/java/DataTypeGenerator.kt

```diff
@@ -255,10 +255,19 @@ abstract class BaseDataTypeGenerator(
             javaType.addAnnotation(disableJsonTypeInfoAnnotation())
         }
 
+        val setterMethodBuilder = MethodSpec.methodBuilder("getAllFields")
+            .addModifiers(Modifier.PUBLIC)
+            .addStatement("java.util.Map<String, Object> fields = new java.util.HashMap()")
```

Comment:
Java Poet wouldn't autoimport this for me, so I referenced the fully qualified name.